tests/fixtures/java_templates/partials_test/.integration-partials.yaml

```diff
@@ -0,0 +1,10 @@
+integration: |
```

Comment:
I want the semantic of this value coming at the end of the generated cfg file. Can you add "append" or something at the key name?

Reply:
I found rennie's comment: https://github.com/googleapis/synthtool/pull/1876#discussion_r1346451926

Reply:
Changed key to `integration_append`.